pkg/controllers/managementuser/cis/clusterScanHandler.go

```diff
@@ -304,6 +317,16 @@ func (csh *cisScanHandler) Updated(cs *v3.ClusterScan) (runtime.Object, error) {
 		}
 		v32.ClusterScanConditionCompleted.True(cs)
 		v32.ClusterScanConditionAlerted.Unknown(cs)
+	} else if v32.ClusterScanConditionFailed.IsTrue(cs) {
+		cluster, err := csh.clusterLister.Get("", csh.clusterNamespace)
+		if err != nil {
+			return nil, fmt.Errorf("cisScanHandler: Updated: error getting cluster %v", err)
+		}
+		updatedCluster := cluster.DeepCopy()
+		updatedCluster.Status.CurrentCisRunName = ""
+		if _, err := csh.clusterClient.Update(updatedCluster); err != nil {
+			return nil, fmt.Errorf("cisScanHandler: Updated: failed to update cluster about CIS scan completion")
```

Comment:
Include the error message from the update call

Reply:
done